src/plugins/content/accessibility/keyNavigation/strategies/stimulusNavigation.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 Open Assessment Technologies SA ;
+ */
+
+import $ from 'jquery';
+import keyNavigator from 'ui/keyNavigation/navigator';
+import navigableDomElement from 'ui/keyNavigation/navigableDomElement';
+import {
+    setupItemsNavigator
+} from 'taoQtiTest/runner/plugins/content/accessibility/keyNavigation/helpers';
+
+/**
+* The identifier the keyNavigator group
+* @type {String}
+*/
+const groupId = 'stimulus-element-navigation-group';
+
+/**
+ * Key navigator strategy applying on stimulus items with scrollbar.
+ * Navigable item content are body elements with the special class "stimulus-container".
+ * @type {Object} keyNavigationStrategy
+ */
+export default {
+    name: 'stimulus',
+
+    /**
+     * Builds the item navigation strategy.
+     *
+     * @returns {keyNavigationStrategy}
+     */
+    init() {
+        const config = this.getConfig();
+        this.keyNavigators = [];
+
+        const $content = this.getTestRunner().getAreaBroker().getContentArea();
+        $content
+            .find('.stimulus-container')
+            // filter out nodes without scrollbar
+            .filter(function () {
+                return this.scrollHeight > this.clientHeight;
```

Comment:
Well, this might work if the size is set before the keyNavigator is created. Unfortunately this is rarely the case. 
A better approach would be to target all stimulus, then dynamically compute if the element can be focused.

A possibility would be to overload the `isEnabled()` method in the `navigableDomElement` instance when creating the keyNavigator:
```javascript
                const elements = navigableDomElement.createFromDoms($element).map(el => {
                    const oldIsEnabled = el.isEnabled;
                    el.isEnabled = function isEnabled() {
                        if (oldIsEnabled.call(this)) {
                            const node = this.getElement().get(0);
                            return node.scrollHeight > node.clientHeight;
                        }
                        return false;
                    };
                    return el;
                });
                const navigator = keyNavigator({
                    id: `${groupId}-${i}`,
                    elements,
                    group: $element,
                    propagateTab: false
                });
```